packages/blitz/src/bin/cli.ts

```diff
@@ -0,0 +1,61 @@
+import * as fs from 'fs'
+import * as path from 'path'
+const resolveGlobal = require('resolve-global')
+const pkgDir = require('pkg-dir')
+
+let usageType: 'local' | 'monorepo' | 'global' | 'global-linked'
+
+const localCLIPkgPath = path.resolve(process.cwd(), 'node_modules', '@blitzjs/cli')
+const monorepoCLIPkgPath = path.resolve(process.cwd(), '../..', 'node_modules', '@blitzjs/cli')
+const globalCLIPkgPath = resolveGlobal.silent('@blitzjs/cli') as string
+const globalLinkedCLIPkgPath = path.resolve(pkgDir.sync(__dirname), '../cli')
+
+function getBlitzPkgJsonPath() {
+  switch (usageType) {
+    case 'local':
+      return path.join(process.cwd(), 'node_modules/blitz/package.json')
+    case 'monorepo':
+      return path.join(process.cwd(), '../../node_modules/blitz/package.json')
+    case 'global':
+      return path.join(resolveGlobal.silent('blitz') || '', 'package.json')
+    case 'global-linked':
+      return path.join(pkgDir.sync(__dirname), 'package.json')
+  }
+}
+
+let pkgPath
+if (fs.existsSync(localCLIPkgPath)) {
+  usageType = 'local'
+  pkgPath = localCLIPkgPath
+} else if (fs.existsSync(monorepoCLIPkgPath)) {
+  usageType = 'monorepo'
+  pkgPath = monorepoCLIPkgPath
+} else if (fs.existsSync(globalCLIPkgPath)) {
+  usageType = 'global'
+  pkgPath = globalCLIPkgPath
+} else {
+  usageType = 'global-linked'
+  pkgPath = globalLinkedCLIPkgPath
+}
+
+// TODO: remove
+console.log('DEBUG:', usageType)
+console.log('Path:', pkgPath)
```

Comment:
When do we remove this?

Reply:
After I publish everything NPM and see if it's working right